facebook_downloader/config.py

```diff
@@ -42,3 +42,8 @@ def access_token() -> str:
 def redownload_window() -> str:
     """The number of days for which the performance data will be redownloaded"""
     return '28'
+
+
+def target_accounts() -> str:
+    """The accounts to download, comma separated, if empty each available account will be tried"""
+    return ''
```

Comment:
Why not a `list[str]`?

Reply:
Because it would not be available in Python 3.5 and the CLI value is a string in any case.

Reply:
I looked for a way to make it backward compatible (something like importing `typing.List` in a try-catch) but didn't find anything nice

Reply:
petty, it also seems that click is not able to return list of strings :-(

Anyway: isn't mara py3.6+ only?

Reply:
It is, but I think some people are only using the downloaders and a less recent Python version, the latest Ubuntu LTS, for example, has Python 3.5

It would be interesting to have stats on the usage of the module